.goreleaser.yaml

```diff
@@ -13,28 +13,10 @@ builds:
       - CGO_ENABLED=0
     goos:
       - linux
-    goarch:
-      - amd64
-      - arm64
-    # Set the modified timestamp on the output binary to the git timestamp (to ensure a reproducible build)
```

Comment:
does this somehow lose the reproducible build now? does it matter?

Reply:
I don't think anything was deleted here. We had two blocks that were doing the builds for windows/linux and I just zipped them into one. `mod_timestamp` is still in the file. `id: syft-win` was removed and everything was consolodated under `id: syft`